js/app/src/api_docs/ApiDocs.svelte

```diff
@@ -79,9 +79,13 @@
 
 	let js_info: Record<string, any>;
 
-	get_info().then((data) => (info = data));
+	get_info().then((data) => {
```

Comment:
Unrelated but I think it'd be nice if we fixed the url for deployed spaces. I think there's an issue open

<img width="1029" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/2aa8c87d-82ec-4045-bb4e-0d0ed5780482">

Reply:
Yeah I was thinking about fixing this, will do so in a follow-up PR